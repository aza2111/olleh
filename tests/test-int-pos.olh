//test-int-pos: operations on ints

int i = 2 + 1;
int j = 3 - 4;
print j; //-1

print i - k;

print 54 % 17; //3
print i * j;
print i * 7;

print 4 / 3;

int x = j + i - -9 * 7 + 65 / 34; //-1 + 3 - -63 + 1 = 66
print x;
