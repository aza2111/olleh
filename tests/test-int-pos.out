-1
4
3
-3
21
1
66
